packages/repository/test/integration/repositories/relation.factory.integration.ts

```diff
@@ -86,17 +147,54 @@ describe('HasMany relation', () => {
     });
   }
 
+  class Review extends Entity {
+    id: number;
+    description: string;
+    authorId: number;
+    // note(shimks): a review shouldn't be liked by only one customer,
+    // but for the sake of testing only one customer is allowed to like a review
+    likedId: number;
```

Comment:
May instead of "liking a review", we could use "approving a review"? I think it's more natural for a review to have a single approver.

Not a big deal, your current test-data design can stay as-is.